modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -212,8 +200,20 @@ class Instrument_Manager extends \NDB_Menu_Filter
         // silently. Otherwise, it will return the exit code and error
         // messsage from MySQL. This will be stored in $result and
         // logged via LorisException.
-        $table_name = \NDB_BVL_Instrument::factory($instrument, '', '')->table;
-        $db_config  = $this->factory->config()->getSetting('database');
+        try {
+            $table_name = \NDB_BVL_Instrument::factory($instrument, '', '')->table;
+        } catch (\ConfigurationException $e) {
+            error_log(
+                "Cannot populate certification tables for uploaded " .
```

Comment:
I don't understand this error message.. what certification tables?

Reply:
It was for the certification/training module. You just deleted that though right?

Reply:
What was for the certification/training module? This line/error message is in the instrument manager, where it looks like it's just generally trying to install the instrument table..

Reply:
Yes but when installing an instrument, if you had the `EnableCertification` setting enabled in config.xml, then the instrument_manager does extra steps to set up certification on an instrument.

Reply:
a \ConfigurationException is a general error that's thrown any time a config setting is used that hasn't been defined in the config table and should be a relatively loud fatal error, I still don't see why NDB_BVL_Instrument::factory throwing an exception is assumed to be a config exception.

Reply:
@johnsaigle ping

Reply:
You're right this is probably at the wrong level of abstraction. The config section happened in a deeper function within the Instrument class, I believe. Can't remember the exact call stack.

Anyway I am going to revert this change since you deleted the training module. Since it doesn't exist anymore, the error shouldn't occur.

Reply:
I cleaned up the leftover training module-releated code that was cuasing my error in #4696